README.md

```diff
@@ -161,6 +161,16 @@ To learn more about AoT compilation, visit [this section of Angular documentatio
 
 Prior to [Angular 2.2.0](https://github.com/angular/angular/blob/master/CHANGELOG.md#220-upgrade-firebooster-2016-11-14), `enableDebugTools()` would clobber `ng.probe`, which breaks Augury. Prior to that version, [this workaround](https://github.com/AngularClass/angular2-webpack-starter/blob/dbb7d10e6e84b8e88116d957f0047b422ab807c1/src/app/environment.ts#L28...L36) will circumvent the issue.
 
+## Support for Angular 9+
+
+Due to some Angular 9 debug API updates, Augury cannot support ngModule or router tree parsing for the moment. As a workaround, disable ivy to be able to see missing Augury tabs by adding this config to your project's `tsconfig.app.json` file : 
```

Comment:
+ "Due to some Angular 9 debug API updates." This is unnecessary information. Remove
+ "Augury cannot support ngModule or router tree parsing for the moment." Rephrase to emphasize that Augury is unable to provide these features for Angular 9 applications that are using Ivy.
+ "disable ivy to be able to see missing Augury tabs" Not specific enough as to what functionality is missing.

Additionally, It should be made very clear that disabling ivy is not recommended, but rather is only a work around for this particular problem. Ivy is enabled by default in new releases. Its use moving forward is encouraged in all new Angular applications.